dotcom-rendering/src/client/userFeatures/cookies/userBenefitsExpiry.ts

```diff
@@ -0,0 +1,17 @@
+import { getCookie } from '@guardian/libs';
+
+export const USER_BENEFITS_EXPIRY_COOKIE = 'gu_user_features_expiry';
```

Comment:
I'm going to change the name of this cookie to `gu_user_benefits_expiry` at some point but I'm waiting for the go ahead from Transparency and Consent before doing that.